src/systems/si/include/units/isq/si/thermodynamic_temperature.h

```diff
@@ -75,7 +146,55 @@ using namespace thermodynamic_temperature_references;
 namespace units::aliases::isq::si::inline thermodynamic_temperature {
 
 template<Representation Rep = double> using K = units::isq::si::thermodynamic_temperature<units::isq::si::kelvin, Rep>;
+template<Representation Rep = double> using mK = units::isq::si::thermodynamic_temperature<units::isq::si::millikelvin, Rep>;
+template<Representation Rep = double> using deg_C = units::isq::si::thermodynamic_temperature<units::isq::si::degree_celsius, Rep>;
 
 }  // namespace units::aliases::isq::si::inline thermodynamic_temperature
 
+
+namespace units {
+
+/** \brief Take a quantity of in a standard conventional temperature unit and re-interpret it as a quantity point
+ *
+ * A \c quantity represents a "difference" between "points" along a "dimension" scale. However, colloquially, temperature
+ * units are double-used to also represent a "point" directly, to be interpreted as a difference to an implicit reference point.
+ *
+ * This function performs this tasks of identifying and adding that implicit reference point.
+ *
+ * WARNING: The returned value's physical meaning depends on the unit of the input quantity, which has no physical relevance on it's own.
+ * That is, 1 Kelvin and 1 degree Celsius are physically equivalent quantities (as differences between thermodynamic temperatures),
+ * yet the returned temperature points are physically different. This is in stark contrast to the usual behaviour of
+ * this library, where unit conversions. For example, for the following holds:
```

Comment:
```suggestion
 * this library, where unit converts. For example, for the following holds:
```
Is this what you meant?